src/components/Modeler.vue

```diff
@@ -42,29 +42,42 @@
           <span class="btn btn-sm btn-secondary scale-value">{{ Math.round(scale*100) }}%</span>
         </div>
 
-        <button class="btn btn-sm btn-secondary mini-map-btn ml-auto" data-test="mini-map-btn" @click="miniMapOpen = !miniMapOpen">
-          <font-awesome-icon v-if="miniMapOpen" :icon="minusIcon" />
-          <font-awesome-icon v-else :icon="mapIcon" />
-        </button>
+        <div class="btn-group btn-group-sm mr-2" role="group" aria-label="Third group">
+          <button class="btn btn-sm btn-secondary ml-auto" data-test="panels-btn" @click="panelsCompressed = !panelsCompressed">
+            <font-awesome-icon v-if="panelsCompressed" :icon="expandIcon" />
+            <font-awesome-icon v-else :icon="compressIcon" />
```

Comment:
These two could be combined into one element, no?
```html
<font-awesome-icon :icon="panelsCompressed  ? expandIcon : compressIcon" />
```

Reply:
In fairness, I think this was copied from the lines 52 and 53 below. Maybe that could be edited too to use a ternary operator?